tests/all/volumes.t

```diff
@@ -158,6 +159,15 @@ USE_NODE nodes[0]
 TEST "kadalu volume stop DEV/vol14 --mode=script"
 TEST "kadalu volume delete DEV/vol14 --mode=script"
 
+# Tests for Backup & Restore
+TEST "kadalu volume create DEV/vol19 server1:/exports/vol19/s1 server2:/exports/vol19/s2 server3:/exports/vol19/s3"
+USE_NODE nodes[0]
+TEST "kadalu backup backup1"
+TEST "kadalu restore backup1 --mode=script"
```

Comment:
Before calling restore, (Just to make sure restore works and not using the old Db and previously running process.

- Stop the Mgr. `systemctl stop kadalu-mgr`
- Delete `/var/lib/kadalu/meta` and `/var/lib/kadalu/info`
- Start the `kadalu-mgr`
- Now call restore

Reply:
Ack to all reviews. Done.